### ISSUE TITLE
Ignore/Exclude Steam shared Redist. Directory

### ISSUE THREAD

**Issue by cvxx7q** [Codeusa/SteamCleaner#74]
Hi there,

i have small issue, that when running steam cleaner it deletes the Shared Redist' Folder steam is now using to consolidate all the Redist' into 1 directory vs every game having its own copy

"~\Steam\steamapps\common\Steamworks Shared"

![image](https://user-images.githubusercontent.com/50063677/56857600-f1014c00-69b2-11e9-816c-e45832dda47f.png)

this folder may be beneficial to keep since it serves many games with one set of redist, older games may not utilise this folder however and can be cleaned up as per what this tool is used for

**Comment by Zorrototo** [comment]
Yes this folder is absolutely a NO for deletion as it is common now to ALL your games in Steam library which render this tool almost/completely useless for Steam when 100% of Steam games will have shifted to this new distribution system.

Funny I commented about that yesterday lol https://github.com/Codeusa/SteamCleaner/issues/71#issuecomment-487250246
